.github/ISSUE_TEMPLATE.md

```diff
@@ -29,6 +29,10 @@ https://github.com/nvaccess/nvda/wiki/Github-issue-template-explanation-and-exam
 #### Other information about your system:
 
 ### Other questions
-#### Does the issue still occur after restarting your PC?
+#### Does the issue still occur after restarting your machine?
```

Comment:
I think I would prefer "computer" rather than machine which may not be so obvious for less technically inclined people. Alternatives I can think of:
- "PC" (note, personally I think this applies to laptop PC and desktop PC, as an acronym for Personal Computer)
- "computer"
- "system"
- "device"
- "machine"